docroot/modules/custom/user_mail/src/Plugin/QueueWorker/WelcomeMail.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Drupal\user_mail\Plugin\QueueWorker;
+
+use Drupal\Core\Entity\EntityTypeManager;
+use Drupal\Core\Mail\MailManager;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Queue\QueueWorkerBase;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Defines 'user_mail_welcomemail' queue worker.
+ *
+ * @QueueWorker(
+ *   id = "user_mail_welcomemail",
+ *   title = @Translation("WelcomeMail"),
+ *   cron = {"time" = 60}
+ * )
+ */
+class WelcomeMail extends QueueWorkerBase implements ContainerFactoryPluginInterface {
+
+  /**
+   * Mail manager for sending mail.
+   *
+   * @var \Drupal\Core\Mail\MailManager
+   */
+  private $mailManager;
+
+  /**
+   * EntityManager Object for loading User.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManager
+   */
+  private $entityTypeManager;
+
+  /**
+   * {@inheritDoc}
+   *
+   * @param \Drupal\Core\Mail\MailManager $mailManager
+   *   For sending mail.
+   * @param \Drupal\Core\Entity\EntityTypeManager $entityTypeManager
+   *   For loading user object.
+   */
+  public function __construct(
+    MailManager $mailManager,
+    EntityTypeManager $entityTypeManager
+  ) {
+    $this->mailManager = $mailManager;
+    $this->entityTypeManager = $entityTypeManager;
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  public static function create(
+    ContainerInterface $container,
+    array $configuration,
+    $plugin_id,
+    $plugin_definition
+  ) {
+    $mailManager = \Drupal::service('plugin.manager.mail');
+    $entityTypeManager = \Drupal::service('entity_type.manager');
+    return new static($mailManager, $entityTypeManager);
```

Comment:
this is not correct way to instantiate object via factory method. Pls check examples in core.